### PR TITLE
Edit address

### PR DESCRIPTION
**Pull request by solid-swan**
I want to merge my branch to the main

### CODE REVIEW

**Review by JoycesCodes — APPROVED** [review]
looks good

File: src/components/addresses/AddressEdit.js [hunk 1]
```diff
@@ -1,9 +1,129 @@
...
+                </Card.Text>
+              </Card.Body>
+            </Card>
+            <Button>Submit</Button>
+            <Button>Clear</Button>
+            {/* <Card>
+              <Card.Text>
+                <h2>Order History</h2>
+              </Card.Text> */}
+            {/* </Card> */}
```
Comment (kchia) [inline src/components/addresses/AddressEdit.js:118]: Remove unused code.

File: src/components/addresses/AddressEdit.js [hunk 2]
```diff
@@ -1,9 +1,129 @@
...
+        </div>
+      </section>
+    );
   }
 }
 
 export default AddressEdit;
+
+// to access this page
+// http://localhost:3000/addresses/1/edit
```
Comment (kchia) [inline src/components/addresses/AddressEdit.js:129]: Remove unused code.

File: src/components/addresses/AddressEdit.js [hunk 3]
```diff
@@ -1,9 +1,129 @@
...
+          state: address.state,
+          zip: address.zip,
+        })
+      )
+      .catch(console.error);
+  }
+  // to access different address IDs
+  // view(id) {
+  //   return http.get(`/addresses/${id}`);
+  // }
```
Comment (kchia) [inline src/components/addresses/AddressEdit.js:36]: Remove unused code.

**Pull request merged**